assets/stylesheets/components/_paginator.scss

```
@@ -0,0 +1,40 @@
+.paginator {
+  $self: &;
+
+  @apply w-full justify-center relative p-2 z-0 inline-flex -space-x-px sm:p-4 sm:pt-2;
+
+  &__page {
+    @apply bg-white border-gray-300 relative inline-flex items-center p-2 border shadow-sm text-sm font-medium hover:bg-gray-50;
+
+    #{$self}__page-text, #{$self}__page-link {
+      @apply text-gray-500 text-center;
+
+      min-width: 20px;
+      min-height: 16px;
```

```suggestion
      @apply text-gray-500 text-center;
      min-width: 20px;
      min-height: 16px;
```

I think we should use new line to separate between element.